packages/story-editor/src/components/panels/design/textStyle/karma/textStyle.other.karma.js

```diff
@@ -143,6 +143,11 @@ describe('Text Style Panel', () => {
               currentPage: { elements },
             },
           } = await fixture.renderHook(() => useStory());
+
+          if (!elements) {
+            throw new Error('story elements not ready');
+          }
```

Comment:
just an extra check in the outermost waitFor to ensure correct repolling. Keep running into timeout errors locally, so hopefully this helps.